### PR TITLE
Improve catalog creation methods name

### PR DESCRIPTION
**Pull request by kokosing**

### CODE REVIEW

File: presto-main/src/main/java/io/prestosql/connector/ConnectorManager.java [hunk 1]
```diff
@@ -168,15 +168,15 @@ public synchronized void addConnectorFactory(ConnectorFactory connectorFactory)
         handleResolver.addConnectorName(connectorFactory.getName(), connectorFactory.getHandleResolver());
     }
 
-    public synchronized CatalogName createConnection(String catalogName, String connectorName, Map<String, String> properties)
+    public synchronized CatalogName addCatalog(String catalogName, String connectorName, Map<String, String> properties)
```
Comment (findepi) [inline presto-main/src/main/java/io/prestosql/connector/ConnectorManager.java]: `createCatalog`?

Comment (kokosing) [reply presto-main/src/main/java/io/prestosql/connector/ConnectorManager.java]: Sounds good to me. Updated.

File: presto-main/src/main/java/io/prestosql/connector/ConnectorManager.java [hunk 2]
```diff
@@ -187,12 +187,12 @@ private synchronized CatalogName createConnection(String catalogName, ConnectorF
...
         checkState(!connectors.containsKey(catalog), "Catalog '%s' already exists", catalog);
 
-        addCatalogConnector(catalog, connectorFactory, properties);
+        createCatalog(catalog, connectorFactory, properties);
 
         return catalog;
     }
 
-    private synchronized void addCatalogConnector(CatalogName catalogName, ConnectorFactory factory, Map<String, String> properties)
+    private synchronized void createCatalog(CatalogName catalogName, ConnectorFactory factory, Map<String, String> properties)
```
Comment (findepi) [inline presto-main/src/main/java/io/prestosql/connector/ConnectorManager.java]: `createCatalogConnector` ?

(it wasn't an overload before)

Comment (kokosing) [reply presto-main/src/main/java/io/prestosql/connector/ConnectorManager.java]: CatalogConnector? This sounds like a butter made from milk.

**Pull request merged**